charts/router/templates/router-clusterrole.yaml

```diff
@@ -0,0 +1,22 @@
+{{ if .Capabilities.APIVersions.Has (include "rbacApiVersion" .) }}
+kind: ClusterRole
+apiVersion: {{ template "rbacApiVersion" . }}
+metadata:
+  name: deis:deis-router
+  labels:
+    app: deis-router
+    heritage: deis
+rules:
+- apiGroups: ["extensions", "apps"]
+  resources: ["deployments"]
+  verbs: ["get", "list", "watch"]
+- apiGroups: [""]
+  resources: ["services"]
+  verbs: ["get", "list", "watch"]
+- apiGroups: [""]
+  resources: ["secrets"]
+  verbs: ["get", "list"]
```

Comment:
Secrets are retrieved by name-- i.e. a get-- so list shouldn't be needed. Could you please try removing list here and verify that the router still works?